ui/handle_input.go

```diff
@@ -155,3 +174,78 @@ func (win *Window) resetPasswords() {
 	win.outputs.matchSpending.HintColor = win.theme.Color.InvText
 	win.inputs.matchSpending.SetText("")
 }
+
+func (win *Window) validateSeeds() string {
+	text := ""
+
+	for i, editor := range win.inputs.seeds {
+		if editor.Text() == "" {
+			win.outputs.seeds[i].HintColor = win.theme.Color.Danger
+			return ""
+		}
+
+		text += editor.Text() + " "
+	}
+
+	return text
+}
+
+func (win *Window) resetSeeds() {
+	for i := 0; i < len(win.inputs.seeds); i++ {
+		win.inputs.seeds[i].SetText("")
+	}
+}
+
+func (win *Window) editorSeedsEventsHandler() {
+	for i, editor := range win.inputs.seeds {
+		if editor.Focused() &&
+			(win.combined.seedEditorsHandlerIndex != i || strings.Trim(editor.Text(), " ") == "") {
+			win.combined.seedEditorsHandlerIndex = -1
+		}
+
+		for _, e := range editor.Events(win.gtx) {
+			switch e.(type) {
+			case widget.ChangeEvent:
+				win.combined.seedEditorsHandlerIndex = i
+			case widget.SubmitEvent:
+				if i < len(win.inputs.seeds)-1 {
+					win.inputs.seeds[i+1].Focus()
+				}
+			}
+		}
+	}
+}
+
+func (win *Window) onSuggestionSeedsClicked() {
+	for i := 0; i < len(win.combined.seedsSuggestionsBtn); i++ {
+		if win.combined.seedsSuggestionsBtn[i].Clicked(win.gtx) {
+			win.inputs.seeds[win.combined.seedEditorsHandlerIndex].SetText(win.combined.seedsSuggestions[i])
+			win.inputs.seeds[win.combined.seedEditorsHandlerIndex].Move(len(win.combined.seedsSuggestions[i]))
+			win.combined.seedsSuggestions = nil
+
+			if win.combined.seedEditorsHandlerIndex < len(win.inputs.seeds)-1 {
+				win.inputs.seeds[win.combined.seedEditorsHandlerIndex+1].Focus()
+			}
+		}
+	}
+}
+
+// KeysEventsHandler handlers all key events when typing editor, if Tab pressed will putting first word
+// from the list of suggestion to the editor
+func (win *Window) KeysEventsHandler() {
```

Comment:
This should take the key event as an argument since it is only used within this scope.